packages/localize/src/cli.ts

```diff
@@ -6,24 +6,40 @@
 
 import * as path from 'path';
 import * as minimist from 'minimist';
-
-import {programFromTsConfig, printDiagnostics} from './typescript';
-import {extractMessagesFromProgram} from './program-analysis';
-import {runtimeOutput} from './outputters/runtime';
-import {transformOutput} from './outputters/transform';
-import {makeFormatter} from './formatters';
-import {
-  ProgramMessage,
-  Message,
-  validateLocalizedPlaceholders,
-} from './messages';
-import {KnownError, throwUnreachable} from './error';
+import {KnownError, unreachable} from './error';
 import {Config, readConfigFileAndWriteSchema} from './config';
-import {Locale} from './locales';
+import {LitLocalizer} from './index.js';
+import {printDiagnostics} from './typescript.js';
+import {TransformLitLocalizer, TransformOutputConfig} from './modes/transform';
+import {RuntimeLitLocalizer, RuntimeOutputConfig} from './modes/runtime';
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 require('source-map-support').install();
 
+const usage = `
+Usage: lit-localize [--config=lit-localize.json] COMMAND
+
+Commands:
+  extract     Extract messages from source files
+  build       Build your project
+
+Options:
+  --help      Display this help message.
+  --config    Path to JSON configuration file.
+              Default: ./lit-localize.json
+              See https://github.com/Polymer/lit-html/tree/lit-next/packages/localize#readme for details.
+`;
+
+const validCommands = ['build', 'extract'] as const;
+type Command = typeof validCommands[number];
+const isValidCommand = (str: string): str is Command =>
```

Comment:
Minor & optional: consider just isCommand rather than isValidCommand

isCommand has a natural relationship to the type relationship `str is Command`

isValidCommand suggests more, like it might throw, or do validation beyond checking whether it's in the type